gneiss/plot/_dendrogram.py

```diff
@@ -0,0 +1,235 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+"""Drawing trees.
+
+Draws horizontal trees where the vertical spacing between taxa is
+constant.  Since dy is fixed dendrograms can be either:
+ - square:     dx = distance
+ - not square: dx = max(0, sqrt(distance**2 - dy**2))
+
+Also draws basic unrooted trees.
+
+For drawing trees use either:
+ - UnrootedDendrogram
+
+Note: This is directly ported from pycogent.
+"""
+
+# Future:
+#  - font styles
+#  - orientation switch
+# Layout gets more complicated for rooted tree styles if dy is allowed to vary,
+# and constant-y is suitable for placing alongside a sequence alignment anyway.
+from skbio import TreeNode
+import pandas as pd
+import numpy
+import abc
+
+
+class Dendrogram(TreeNode):
+    # One of these for each tree edge.  Extra attributes:
+    #    depth - distance from root to bottom of edge
```

Comment:
Not sure.  I removed that documentation, since it appears to be outdated and doesn't reflect the current state of this refactor